msal/broker.py

```diff
@@ -7,8 +7,12 @@
 import time
 import uuid
 
+from .application import __version__
```

Comment:
Better add a new sku.py which contains nothing but `SKU="..."` and `__version__ = "..."`, and then we have both application.py and broker.py (plus __init__.py) do `from .sku import ...`. This way, the application.py and broker.py will not have a circular reference to each other.